chinese/wg-diversity-inclusion/focus-areas/leadership/inclusive-leadership.md

```diff
@@ -1,42 +1,47 @@
 # 包容性领导
 
-问题：项目对多元化领导层的支持如何？
+问题：项目和社区对多元化领导层的支持有多好？
```

Comment:
Since you already updated the description of question, would you please update this question to readme https://github.com/chaoss/translations/blob/main/chinese/wg-diversity-inclusion/focus-areas/leadership/README.md?